packages/ui/react-ui-table/src/model/table-sorting.ts

```diff
@@ -38,13 +34,12 @@ export type SortConfig = { fieldId: string; direction: SortDirection };
  */
 export class TableSorting<T extends BaseTableRow> {
```

Comment:
This sorting module became a whole lot more general purpose now that we only rely on the view.